vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -0,0 +1,257 @@
+---
+title: vCluster Pro air-gapped install
+sidebar_label: Air-gapped
+sidebar_position: 5
+description: Learn how to install vCluster Pro in an air-gapped Kubernetes cluster without the vCluster Platform agent.
+sidebar_class_name: pro
+---
+
+import Flow, { Step } from "@site/src/components/Flow";
+import BasePrerequisites from '../../../platform/_partials/install/base-prerequisites.mdx';
+
+import FipsConfig from '../../_partials/deploy/fips-config.mdx';
+
+# vCluster Pro air-gapped install
+
+This document details the prerequisites and steps to install vCluster Pro into
+an `air-gapped` Kubernetes cluster without the platform agent running on the host
+cluster.
+
+## Prerequisites
+
+<BasePrerequisites />
+  - `docker` (check with `docker version`)
+  - An offline license key for vCluster Pro (provided by Loft)
+- Optionally: private docker registry that the installer computer and the
+air-gapped Kubernetes cluster can access (e.g. x-private-registry:5000 or
+gcr.io/x-team)
+- vCluster Pro syncer (control plane `StatefulSet`) pod default resource requirements (may need to be increased based on intended vCluster workloads):
+
+```yaml title="vCluster Pro syncer pod resource requirements"
+  - Requests
+    - memory: 256Mi
+    - cpu: 200m
+    - ephemeral-storage: 400Mi
+  - Limits
+    - memory: 2Gi
+    - ephemeral-storage: 8Gi
+```
+
+### Download vCluster Helm chart
+
+Download https://charts.loft.sh/charts/vcluster-0.20.3.tgz:
```

Comment:
Not sure if this should be a specific image. If so, will it be updated to the latest GA release?